reports/report_details.py

```diff
@@ -1555,7 +1556,7 @@ def add_transnational_to_regions(regions):
             'name': '101',
             'title': 'Covid stories:  reporters, by sex',
             'desc': 'Covid stories:  reporters, by sex',
```

Comment:
```suggestion
            'desc': '',
```
No description in queries sheet.